src/libs/error.ts

```diff
@@ -1,12 +1,17 @@
 import { AnyPromiseFn, GenericPromiseFn, safeCurry } from '@terra-ui-packages/core-utils';
 import _ from 'lodash/fp';
+// import { sessionTimedOutErrorMessage } from 'src/libs/ajax/ajax-common';
 import { notify } from 'src/libs/notifications';
 
 export const reportError = async (title, obj) => {
   console.error(title, obj); // helpful when the notify component fails to render
   // Do not show an error notification when a session times out.
   // Notification for this case is handled elsewhere.
-  if (obj instanceof Error && obj.message === 'Session timed out') {
+  // For some reason making using sessionTimedOutErrorMessage instead of the raw string breaks tests
+  if (
+    obj instanceof Error &&
+    obj.message === 'Session timed out (due to auth token refresh failure or expired refresh token)'
```

Comment:
Since there's logic based on the error message, this should use the constant defined in `ajax-common`.

As is, it's too easy for the error message there and here to get out of sync and break this logic.

Reply:
Fixed!